src/kiwixseeder/entrypoint.py

```diff
@@ -0,0 +1,304 @@
+import argparse
+import signal
+import sys
+from types import FrameType
+
+import humanfriendly
+import qbittorrentapi
+from rich.markdown import (  # pyright: ignore [reportMissingImports]
+    Markdown,  # pyright: ignore [reportUnknownVariableType]
+)
+from rich_argparse import (  # pyright: ignore [reportMissingImports]
+    RichHelpFormatter,  # pyright: ignore [reportUnknownVariableType]
+)
+
+from kiwixseeder.__about__ import __version__
+from kiwixseeder.context import (
+    CLI_NAME,
+    DEFAULT_FILTER_FILESIZES,
+    DEFAULT_KEEP_DURATION,
+    DEFAULT_MAX_STORAGE,
+    DEFAULT_QBT_CONN,
+    Context,
+    QbtConnection,
+    SizeRange,
```

Comment:
`SizeRange` comes from `kiwixseeder.utils` in fact (even if it works)

Reply:
it's incidental (used to be in context)